### DISCUSSION TITLE
Where is the 0.3.2 release of tailwindcss-forms?

### DISCUSSION THREAD

**Discussion by wondering639** [tailwindlabs/tailwindcss#4344] [Help] [answered]
On https://github.com/tailwindlabs/tailwindcss-forms/releases the latest version is 0.3.**1**. On npmjs it is 0.3.**2**:
https://www.npmjs.com/package/@tailwindcss/forms

What's going on here?

**Comment by adamwathan** [comment, marked as answer]
Hey! npm versions are not really directly related to GitHub — npm stores it's own copy of the code when you publish there. So this sort of thing happens a lot if you push a version to npm, then just forget to push that same tag to GitHub. I just ran `git push --tags` locally and now it's there, so I must have forgotten to do it. Nothing really to worry about though, the versions on npm are the source of truth, not GitHub.
